src/main/java/mate/academy/spring/dto/request/UserRegistrationDto.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.spring.dto.request;
+
+import javax.validation.constraints.Size;
+import mate.academy.spring.validator.Email;
+import mate.academy.spring.validator.RepeatPassword;
+
+@RepeatPassword
+public class UserRegistrationDto {
+    @Email
+    private String email;
+    @Size(min = 8)
```

Comment:
According the common mistakes:
`Don’t repeat checks for fields: you must ensure email field not to be null in your custom Validator, so there is no need to put @NotNull for that field as well in Dto class. The same goes for the password field.
`
So I think we can also move size check to the custom validator.